s/stringtemplate4/stringtemplate4_ubuntu_16.04.sh

```diff
@@ -0,0 +1,57 @@
+https://github.com/ppc64le/build-scripts/pull/875
```

Comment:
what is the extra line here ?

Reply:
Have removed its typo. Didn't find any tests.